src/components/recipe.test.tsx

```diff
@@ -2,18 +2,34 @@ import React from "react";
 import { render, screen } from "@testing-library/react";
 import Recipe from "./recipe";
 
+const recipeItem = {
+  name: "Recipe Name",
+  description: "Recipe description",
+  servings: 10,
+  prepTime: "10 minutes",
+  cookTime: "20 minutes",
+  sproutyPie: false,
+  lollysPantry: true,
+};
+
 describe("when rendered", () => {
   it("should have the recipe text", () => {
-    render(<Recipe />);
-    expect(screen.getByText("Cole's Roasted Salsa")).toBeInTheDocument();
-    expect(
-      screen.getByText(
-        "A delicious spicy salsa made with roasted tomatoes and serrano peppers"
-      )
-    ).toBeInTheDocument();
+    render(
+      <Recipe
+        name={recipeItem.name}
```

Comment:
Instead of having the Recipe component having 7 separate "props", you could have a single "prop" recipe, so that you could instead use `recipe={recipeItem}`. You could also ensure that the recipe values matches the `IRecipeItem` TypeScript interface.